components/menu/src/menu/menu.js

```diff
@@ -25,16 +30,19 @@ const Menu = ({ children, className, dataTest, dense }) => (
                 position: relative;
                 width: 100%;
                 margin: 0;
-
                 padding: 0;
                 user-select: none;
             }
+            ul.focused {
+                border: 2px solid ${colors.blue600};
+            }
         `}</style>
     </ul>
 )
 
 Menu.defaultProps = {
     dataTest: 'dhis2-uicore-menulist',
+    isFocused: false,
```

Comment:
I think it would be good to add these also to the typescript types in components/menu/types/index.d.ts